dds/DCPS/TopicImpl.cpp

```diff
@@ -256,9 +256,9 @@ bool TopicImpl::check_data_representation(const DDS::DataRepresentationIdSeq& qo
   // if non 0 length data reader compare both lists for a compatible QoS
   else{
     bool found = false;
-    for (int i = 0; i < qos_ids.length(); i++) {
-      for (int j = 0; j < type_allowed_reprs.length(); j++) {
-        if(qos_ids[i] == type_allowed_reprs[j]) {
+    for (int i = 0; i < qos_ids.length(); ++i) {
+      for (int j = 0; j < type_allowed_reprs.length(); ++j) {
+        if (qos_ids[i] == type_allowed_reprs[j]) {
         found = true;
         break;
```

Comment:
Missing indentation inside the `if` statement.

Reply:
It appears as if this is the case because the if conditional is split across 2 lines. I'll just put it back on one line.